nodejs/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Join us at https://github.com/GoogleCloudPlatform/cloud-code-samples.
+
+## Run the Tests
+
+Some of the samples are tested by a dedicated [Cloud Build](https://cloud.google.com/cloud-build)  configuration.
+
+All sample tests are expected to take the following steps:
```

Comment:
Maybe worth outlining a few principles also:

1) Tests should be isolate from each other
2) Disposability - tests should ensure they dispose all the resources they created upon successful completion